pkg/volume/emptydir/quota.go

```diff
@@ -84,6 +94,11 @@ func (cr *realQuotaCommandRunner) RunApplyQuotaCommand(fsDevice string, quota re
 	return "", stderr.String(), err
 }
 
+func (cr *realQuotaCommandRunner) RunMountOptionsCommand(dir string) (string, string, error) {
+	outBytes, err := exec.Command("mount").Output()
```

Comment:
@pmorie it's returned here and used down in isMountedWithGrpquota. There's an abstraction layer in here where the very tiny piece of code that runs the command itself can be swapped out so I could write pure unit tests that just feed in fake output from the CLI. Everything that parses that output however I wanted to remain covered by the tests.